master/getting-started/kubernetes/installation/gce.md

```diff
@@ -2,140 +2,5 @@
-
-```shell
-gcloud compute firewall-rules create calico-ipip --allow 4 --network "default" --source-ranges "10.128.0.0/9"
-```
-
-You can verify the rule with this command:
-
-```shell
-gcloud compute firewall-rules list
-```
-
-### 1.3 Download the required files
-
-    mkdir cloud-config; cd cloud-config
-    curl -O {{site.url}}{{page.dir | replace: 'installation', 'cloud-config' }}master-config.yaml
-    curl -O {{site.url}}{{page.dir | replace: 'installation', 'cloud-config' }}node-config.yaml
-    cd ..
-
-## 2. Deploy the VMs
-
-Deploy the Kubernetes master node using the following command:
-
-```shell
-gcloud compute instances create \
-  kubernetes-master \
-  --image-project coreos-cloud \
-  --image coreos-stable-1010-6-0-v20160628 \
-  --machine-type n1-standard-1 \
-  --metadata-from-file user-data=cloud-config/master-config.yaml
-```
-
-Deploy at least one worker node using the following command:
-
-```
-gcloud compute instances create \
-  kubernetes-node-1 \
-  --image-project coreos-cloud \
-  --image coreos-stable-1010-6-0-v20160628 \
-  --machine-type n1-standard-1 \
-  --metadata-from-file user-data=cloud-config/node-config.yaml
-```
-
-You should have SSH access to your machines using the following command:
-
-```
-gcloud compute ssh <INSTANCE NAME>
-```
-
-## Configure the Cluster
-
-### 3.1 Configure kubectl
-
-The following steps configure remote kubectl access to your cluster.
-
-Download `kubectl`
-
-```shell
-sudo wget -O /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.5.1/bin/linux/amd64/kubectl
-sudo chmod +x /usr/local/bin/kubectl
-```
-
-The following command sets up SSH forwarding of port 8080 to your master node so that you can run `kubectl` commands on your local machine.
-
-```shell
-gcloud compute ssh kubernetes-master --quiet --ssh-flag="-nNT" --ssh-flag="-L 8080:localhost:8080" &
-```
-
-Verify that you can access the Kubernetes API.  The following command should return a list of Kubernetes nodes.
-
-```shell
-kubectl get nodes
-```
-
->If successful, the above command should output something like this:
-
-```shell
-NAME          STATUS                     AGE
-10.240.0.25   Ready,SchedulingDisabled   6m
-10.240.0.26   Ready                      6m
-```
-
-## 4. Install Addons
-
-{% include {{page.version}}/install-k8s-addons.md %}
-
-## Next Steps
-
-You should now have a fully functioning Kubernetes cluster using Calico for networking.  You're ready to use your cluster.
-
-We recommend you try using [Calico for Kubernetes NetworkPolicy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/tutorials/simple-policy).
-
-[cni-plugin]: https://github.com/projectcalico/cni-plugin
-[coreos-gce]: https://coreos.com/os/docs/latest/booting-on-google-compute-engine.html
-[gcloud-instructions]: https://cloud.google.com/sdk/
+See [this page]({{site.baseurl}}/{{page.version}}/reference/public-cloud/gce)
```

Comment:
From my talking with Turk I believe we expect those pages to go away as much as possible with only stuff that would be directly needed for k8s.  The reason the page was not dropped altogether was to have a way to redirect to the new page.  (Using a redirect_from would not mesh with our current `latest` redirection.)